src/core/ExpandableBlock/ExpandableBlock.test.tsx

```diff
@@ -166,3 +166,16 @@ it('should render small size', () => {
   expect(expandableBlock).toBeTruthy();
   expect(expandableBlock.classList).toContain('iui-small');
 });
+
+it('should render borderless', () => {
+  const { container } = render(
+    <ExpandableBlock title='test title' isBorderless={true}>
+      content
+    </ExpandableBlock>,
+  );
+  const expandableBlock = container.querySelector(
+    '.iui-expandable-block',
+  ) as HTMLElement;
+  expect(expandableBlock).toBeTruthy();
+  expect(expandableBlock.classList).toContain('iui-borderless');
```

Comment:
This can be simplified down to one statement

```suggestion
  expect(container.querySelector('.iui-expandable-block').toHaveClass('iui-borderless');
```

Reply:
This statement has been simplified